appveyor.yml

```diff
@@ -2,37 +2,46 @@ build: false
 shallow_clone: false
 platform: 'x86'
 clone_folder: c:\projects\bake
+
 branches:
   only:
     - master
+
 environment:
   global:
     PHP: "C:/PHP"
+
   matrix:
       - db: 2012
         db_dsn: 'sqlserver://sa:Password12!@.\SQL2012SP1/cakephp?MultipleActiveResultSets=false'
+
 services:
   - mssql2012sp1
+
 init:
   - SET PATH=C:\php\;%PATH%
+
 install:
   - cd c:\
-  - ps: Start-FileDownload 'http://ci.cakephp.org/php.zip'
+  - appveyor DownloadFile http://windows.php.net/downloads/releases/archives/php-5.5.8-nts-Win32-VC11-x86.zip -FileName php.zip
   - 7z x php.zip -oc:\php
+  - appveyor DownloadFile https://dl.dropboxusercontent.com/s/1cdq70jbx8vcf56/sqlsrv_unofficial_3.0.2.2.zip -FileName sqlsrv.zip
```

Comment:
Should we not be using the official 3.2 version available from https://www.microsoft.com/en-us/download/details.aspx?id=20098?